README.md

```diff
@@ -1,4 +1,5 @@
 # kube-rs
+
```

Comment:
Uh, this must have been a markdown linter or something. I can revert these changes if desired.

Reply:
ah, that's no problem, but I see you've fixed it.